sbi/inference/snpe/snpe_base.py

```diff
@@ -369,6 +377,8 @@ def build_posterior(
             sample_with_mcmc=sample_with_mcmc,
             mcmc_method=mcmc_method,
             mcmc_parameters=mcmc_parameters,
+            # Pass device of the net as this is where computations will be performed.
```

Comment:
Not sure if this comment is needed. I'm fine with dropping it. Same for `snle_base` and `snre_base`.